docker/Dockerfile

```diff
@@ -19,7 +19,7 @@ RUN echo 'export PATH="$HOME/.tools:$PATH"' >> $HOME/.profile
 # swift-format
 ARG swiftformat_version=508.0.0
 RUN git clone --branch $swiftformat_version --depth 1 https://github.com/apple/swift-format $HOME/.tools/swift-format-source
-RUN cd $HOME/.tools/swift-format-source && swift build -c release
+RUN cd $HOME/.tools/swift-format-source && swift build -c release -Xswiftc -strict-concurrency=complete
```

Comment:
This is us building `swift-format` for the formatting we do in the CI pipeline. This isn't where we want to be adding this.

This should be added to the `docker-compose.yaml` for the `integration-test` "service".

Reply:
yes but i already figured that out and fixed it 😅 this commit is outdated.

Reply:
Great, and sorry—Github notifications brought me here, didn't spot you'd resolved it already :)